source/features/releases-tab.tsx

```diff
@@ -103,11 +103,16 @@ async function init(): Promise<false | void> {
 			releasesTab.setAttribute('aria-current', 'page');
 		}
 
-		select('.dropdown-divider', repoNavigationBar)!.before(
-			createDropdownItem('Releases', `/${repoUrl}/releases`, {
-				'data-menu-item': 'rgh-releases-item'
-			})
-		);
+		const moreDropdown = select('.dropdown-divider', repoNavigationBar);
```

Comment:
Actually we should just match the logic on line 85, which simply appends it. We only need to slightly modify because we have a divider. Thankfully we have an helper function that does this exactly. I think the only change needed is:

```suggestion
		appendBefore(
			select('.js-responsive-underlinenav .dropdown-menu ul'),
			'.dropdown-divider',

			createDropdownItem('Releases', `/${repoUrl}/releases`, {
				'data-menu-item': 'rgh-releases-item'
			})
		)
```

Reply:
Off topic alert!  Give me a min powering up the computer...

Reply:
Works perfectly!